2022-02-02 17:23:19,877 INFO    MainThread:4109 [wandb_setup.py:_flush():71] setting env: {}
2022-02-02 17:23:19,878 INFO    MainThread:4109 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-02 17:23:19,878 INFO    MainThread:4109 [wandb_init.py:_log_setup():371] Logging user logs to /home/robotlabx/Desktop/DTAI/decnet2022/src/models/wandb/run-20220202_172319-ksqq5rvc/logs/debug.log
2022-02-02 17:23:19,878 INFO    MainThread:4109 [wandb_init.py:_log_setup():372] Logging internal logs to /home/robotlabx/Desktop/DTAI/decnet2022/src/models/wandb/run-20220202_172319-ksqq5rvc/logs/debug-internal.log
2022-02-02 17:23:19,878 INFO    MainThread:4109 [wandb_init.py:init():404] calling init triggers
2022-02-02 17:23:19,878 INFO    MainThread:4109 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-02-02 17:23:19,878 INFO    MainThread:4109 [wandb_init.py:init():460] starting backend
2022-02-02 17:23:19,878 INFO    MainThread:4109 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-02 17:23:19,896 INFO    MainThread:4109 [backend.py:ensure_launched():216] starting backend process...
2022-02-02 17:23:19,909 INFO    MainThread:4109 [backend.py:ensure_launched():222] started backend process with pid: 4165
2022-02-02 17:23:19,910 INFO    MainThread:4109 [wandb_init.py:init():469] backend started and connected
2022-02-02 17:23:19,911 INFO    MainThread:4109 [wandb_init.py:init():533] updated telemetry
2022-02-02 17:23:19,947 INFO    MainThread:4109 [wandb_init.py:init():563] communicating current version
2022-02-02 17:23:20,452 INFO    MainThread:4109 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-02 17:23:20,452 INFO    MainThread:4109 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-02 17:23:20,687 INFO    MainThread:4109 [wandb_init.py:init():606] starting run threads in backend
2022-02-02 17:23:21,883 INFO    MainThread:4109 [wandb_run.py:_console_start():1810] atexit reg
2022-02-02 17:23:21,884 INFO    MainThread:4109 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-02-02 17:23:21,884 INFO    MainThread:4109 [wandb_run.py:_redirect():1689] Redirecting console.
2022-02-02 17:23:21,886 INFO    MainThread:4109 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-02 17:23:21,886 INFO    MainThread:4109 [wandb_init.py:init():633] run started, returning control to user process
2022-02-02 17:23:22,156 INFO    MainThread:4109 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-02-02 17:23:22,156 INFO    MainThread:4109 [wandb_run.py:_restore():1752] restore
2022-02-02 17:23:24,411 INFO    MainThread:4109 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 786
  total_bytes: 786
}

2022-02-02 17:23:24,675 INFO    MainThread:4109 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 786
  total_bytes: 786
}

2022-02-02 17:23:24,779 INFO    MainThread:4109 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 786
  total_bytes: 13160
}

2022-02-02 17:23:24,884 INFO    MainThread:4109 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 786
  total_bytes: 13160
}

2022-02-02 17:23:24,987 INFO    MainThread:4109 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 786
  total_bytes: 13160
}

2022-02-02 17:23:25,092 INFO    MainThread:4109 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13160
  total_bytes: 13160
}

2022-02-02 17:23:25,197 INFO    MainThread:4109 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13160
  total_bytes: 13160
}

2022-02-02 17:23:25,303 INFO    MainThread:4109 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13160
  total_bytes: 13160
}

2022-02-02 17:23:25,407 INFO    MainThread:4109 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13160
  total_bytes: 13160
}

2022-02-02 17:23:25,511 INFO    MainThread:4109 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13160
  total_bytes: 13160
}

2022-02-02 17:23:26,083 INFO    MainThread:4109 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13160
  total_bytes: 13160
}

2022-02-02 17:23:26,617 INFO    MainThread:4109 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13160
  total_bytes: 13160
}
local_info {
}

2022-02-02 17:23:27,686 INFO    MainThread:4109 [wandb_run.py:_append_files():2180] logging synced files
